hs_core/hydroshare/hs_bagit.py

```diff
@@ -43,36 +36,34 @@ def create_bag(resource):
     """
     from . import utils as hs_core_utils
     DATE_FORMAT = "YYYY-MM-DDThh:mm:ssTZD"
+    istorage=IrodsStorage()
 
     dest_prefix = getattr(settings, 'BAGIT_TEMP_LOCATION', '/tmp/hydroshare/')
     bagit_path = os.path.join(dest_prefix, resource.short_id, arrow.get(resource.updated).format("YYYY.MM.DD.HH.mm.ss"))
-    visualization_path = os.path.join(bagit_path, 'visualization')
-    contents_path = os.path.join(bagit_path, 'contents')
 
-    for d in (dest_prefix, bagit_path, visualization_path, contents_path):
+    for d in (dest_prefix, bagit_path):
         try:
             os.makedirs(d)
```

Comment:
Good point. I have added EACCES error handling to raise an exception with exception message. Have pushed the changes to my branch for further review.